### ISSUE TITLE
Freeform: Purging Old Spam Submissions fails.

### ISSUE THREAD

**Issue by bartdigitalpulse** [solspace/craft3-freeform#1666]
### What happened?

Queue is filled with failed "Purging Old Spam Submissions" jobs. Checking the logs I see it fails when trying to remove the uploaded files:

```
Error: craft\services\Elements::deleteElementById(): Argument #1 ($elementId) must be of type int, string given
Location: /vendor/solspace/craft-freeform/packages/plugin/src/Bundles/Fields/Implementations/FileUpload/FileRemovalOnSubmissionDelete.php on line 32
``` 

When I check the submissions these are simple bot submissions with no uploaded file in the asset field. They probably managed to upload an empty string? 

Adding a "check empty" and "is integer" before deleteElementById fixes the issue. 

Example bot submission
![5dxAB4xcoF](https://github.com/user-attachments/assets/05bf3cfb-1c3f-44c9-920f-ef6a484c9b06)

### Errors and Stack Trace (if available)

```shell
#0 /vendor/solspace/craft-freeform/packages/plugin/src/Bundles/Fields/Implementations/FileUpload/FileRemovalOnSubmissionDelete.php(32): craft\services\Elements->deleteElementById()
#1 [internal function]: Solspace\Freeform\Bundles\Fields\Implementations\FileUpload\FileRemovalOnSubmissionDelete->removeSubmissionFiles()
#2 /vendor/yiisoft/yii2/base/Event.php(312): call_user_func()
#3 /vendor/yiisoft/yii2/base/Component.php(650): yii\base\Event::trigger()
#4 /vendor/craftcms/cms/src/base/Element.php(5240): yii\base\Component->trigger()
#5 /vendor/craftcms/cms/src/services/Elements.php(2164): craft\base\Element->afterDelete()
#6 /vendor/solspace/craft-freeform/packages/plugin/src/Services/SubmissionsService.php(393): craft\services\Elements->deleteElement()
#7 /vendor/solspace/craft-freeform/packages/plugin/src/Jobs/PurgeSpamJob.php(14): Solspace\Freeform\Services\SubmissionsService->purgeSubmissions()
#8 /vendor/yiisoft/yii2-queue/src/Queue.php(243): Solspace\Freeform\Jobs\PurgeSpamJob->execute()
#9 /vendor/yiisoft/yii2-queue/src/cli/Queue.php(162): yii\queue\Queue->handleMessage()
#10 /vendor/yiisoft/yii2-queue/src/cli/Command.php(146): yii\queue\cli\Queue->execute()
#11 [internal function]: yii\queue\cli\Command->actionExec()
#12 /vendor/yiisoft/yii2/base/InlineAction.php(57): call_user_func_array()
#13 /vendor/yiisoft/yii2/base/Controller.php(178): yii\base\InlineAction->runWithParams()
#14 /vendor/yiisoft/yii2/console/Controller.php(180): yii\base\Controller->runAction()
#15 /vendor/craftcms/cms/src/console/ControllerTrait.php(90): yii\console\Controller->runAction()
#16 /vendor/yiisoft/yii2/base/Module.php(552): craft\queue\Command->runAction()
#17 /vendor/yiisoft/yii2/console/Application.php(180): yii\base\Module->runAction()
#18 /vendor/craftcms/cms/src/console/Application.php(91): yii\console\Application->runAction()
#19 /vendor/yiisoft/yii2/console/Application.php(147): craft\console\Application->runAction()
#20 /vendor/craftcms/cms/src/console/Application.php(122): yii\console\Application->handleRequest()
#21 /vendor/yiisoft/yii2/base/Application.php(384): craft\console\Application->handleRequest()
#22 /craft(13): yii\base\Application->run()
#23 {main}
```

### How can we reproduce this?

1. Spambot submission in a form with asset field
2. Bot manages to submit empty string
3. Set purge submissions by x days

### Freeform Edition

Pro

### Freeform Version

5.7.4

### Craft Version

Craft Pro 4.13.3

### When did this issue start?

Unsure

**Comment by kjmartens** [comment]
Sorry for the delay and the trouble @bartdigitalpulse,

I will have a developer check into this shortly. 🙂

**Comment by kjmartens** [comment]
This is now resolved in Freeform 5.8.2+ 🙂
